apps/dex/src/compounds/ManageLiquidityModal/RemoveLiquidityForm.tsx

```diff
@@ -1,15 +1,18 @@
 import { Button, Fieldset, Input, RacetrackSpinnerIcon } from "@sifchain/ui";
 import BigNumber from "bignumber.js";
 import { clamp } from "rambda";
-import { ChangeEventHandler, FormEventHandler, useCallback, useMemo } from "react";
+import { ChangeEventHandler, FormEvent, FormEventHandler, useCallback, useMemo } from "react";
 import { useLiquidityProviderQuery } from "~/domains/clp";
 import useUnlockLiquidity from "~/domains/clp/formHooks/useUnlockLiquidity";
 import useUnlockLiquidityMutation from "~/domains/clp/hooks/useUnlockLiquidityMutation";
 import { useTokenRegistryQuery } from "~/domains/tokenRegistry";
 import type { ManageLiquidityModalProps } from "./types";
+import {useRewardParams} from "~/domains/clp/hooks/useRewardParams";
+import useRemoveLiquidityMutation from "~/domains/clp/hooks/useRemoveLiquidityMutation";
 
 const UnlockLiquidityForm = (props: ManageLiquidityModalProps) => {
   const liquidityProviderQuery = useLiquidityProviderQuery(props.denom);
+  const rewardParams = useRewardParams();
```

Comment:
@alanrsoares this is returning undefined, can you see any issues?

Reply:
the client might be outdated